src/components/nav-bar/index.tsx

```diff
@@ -1,157 +1,147 @@
+          label: 'Donations',
+        },
+        { href: '/donors', icon: <HandshakeRounded />, label: 'Donors' },
+      ],
+    },
+    {
+      title: 'Inventory',
+      icon: <Inventory />,
+      buttons: [
+        { href: '/donationItem', icon: <Inventory />, label: 'Inventory' },
+        { href: '/item/add', icon: <AddBox />, label: 'Add Evaluation' },
+        { href: '/item', icon: <Assignment />, label: 'Evaluations' },
+      ],
+    },
+    {
+      title: 'Reporting & Settings',
+      icon: <MiscellaneousServices />,
+      buttons: [
+        { href: '/mailMerge', icon: <Email />, label: 'Emails' },
+        { href: '/user', icon: <Group />, label: 'Users' },
+        { href: '#', icon: <Equalizer />, label: 'Reports' },
+      ],
+    },
+  ];
+
   return (
-    <>
-      <Box height={'100vh'}>
-        <Box display={'flex'} justifyContent={'center'} p={2}>
+    <Box
+      height="100vh"
+      display="flex"
+      flexDirection="column"
+      width="33vh" // Fix width of the navbar
+      overflow="hidden" // Prevent any expanding overflow
+      borderRight="1px solid #ddd"
+    >
+      {/* Logo Section */}
+      <Box display="flex" justifyContent="center">
+        <Button href="/">
           <Image
-            className="rounded-md"
             alt="logo"
             src="/cropped-MOH-Logo-768x393.png"
-            width="180"
-            height="100"
-            // style={{ marginRight: '25px' }} // Add spacing between the image and links
+            width="220"
+            height="107"
+            className="rounded-md"
           />
-        </Box>
-
-        <Box display={'flex'} justifyContent={'flex-start'} p={2}>
-          <Button
-            href="./"
-            startIcon={<HomeIcon />}
-            style={{
-              color: '#ff8a65',
-              fontSize: '25px',
-              textTransform: 'none',
-            }}
-          >
-            Home
-          </Button>
-        </Box>
-
-        <Box display={'flex'} justifyContent={'flex-start'} p={2}>
-          <Button
-            href="/donation/add"
-            startIcon={<AddCircleIcon />}
-            style={{
-              color: '#ff8a65',
-              fontSize: '25px',
-              textTransform: 'none',
-            }}
-          >
-            Add Donation
-          </Button>
-        </Box>
-
-        <Box display={'flex'} justifyContent={'flex-start'} p={2}>
-          <Button
-            href="/donation"
-            startIcon={<VolunteerActivismIcon />}
-            style={{
-              color: '#ff8a65',
-              fontSize: '25px',
-              textTransform: 'none',
-            }}
-          >
-            Donations
-          </Button>
-        </Box>
+        </Button>
+      </Box>
 
-        <Box display={'flex'} justifyContent={'flex-start'} p={2}>
+      {/* Button Groups */}
+      {sections.map((section, index) => (
+        <Box key={index} width="100%">
```

Comment:
This .map is generating an error. Fix by adding a key prop like what is done with the second .map further on down in this file

![image](https://github.com/user-attachments/assets/267149d8-6914-492c-8eeb-70f6c20c21d7)

Reply:
I fixed as your recommendation but it still does not work.
However, I don't think the issue comes from my Nav-bar as the error does not happen when I clicked to other pages. It only happens when I clicked to the AddDonationView page.

Reply:
Gotcha, we can not worry about that for now